### PR TITLE
Add solution to Sublist problem.

### PR DESCRIPTION
**Pull request by rudresh4**
## Why?
Solve unimplemented Sublist problem for Ruby.

## What?
Sublist problem description: https://github.com/exercism/problem-specifications/blob/master/exercises/sublist/description.md

## How Has This Been Tested?
rake test
rake test:sublist

## Types of changes
- [x] Add Sublist solution
- [x] Add Sublist test case
- [x] Add Sublist generator
- [x] Add Sublist README
- [x] Update config.json

### CODE REVIEW

File: exercises/sublist/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+A = [3, 4], B = [1, 2, 3, 4, 5], A is a sublist of B
+A = [1, 2, 3], B = [1, 2, 3], A is equal to B
+A = [1, 2, 3, 4, 5], B = [2, 3, 4], A is a superlist of B
+A = [1, 2, 4], B = [1, 2, 3, 4, 5], A is not a superlist of, sublist of or equal to B
+
+* * * *
+
+For installation and learning resources, refer to the
+[exercism help page](http://exercism.io/languages/ruby).
+
```
Comment (Insti) [inline exercises/sublist/README.md]: Should also include instructions about how to run the tests.

**Comment by Insti** [comment]
This is a great PR, thanks.
It includes everything we need.

I love that you included a link to the canonical data 👍 





I'd like us to implement this problem in a property based manner, see: https://github.com/exercism/problem-specifications/pull/424 for how it was done with triangles.
This might be able to be done without a change to the canonical data.

File: exercises/sublist/README.md (hunk 1 above)
Comment (rudresh4) [reply exercises/sublist/README.md]: I solved this problem once and for all by autogenerating READMEs: https://github.com/exercism/ruby/pull/721

**Comment by rudresh4** [comment]
I'd rather have the changes made to canonical data rather than make workarounds for it.

**Comment by Insti** [comment]
In which case we'll need to submit an issue/PR to the https://github.com/exercism/problem-specifications repository about that.

**Pull request closed without merging**